ui/v2.5/src/components/Settings/SettingsConfigurationPanel.tsx

```diff
@@ -549,7 +550,7 @@ export const SettingsConfigurationPanel: React.FC = () => {
           </Form.Text>
         </Form.Group>
         <Form.Group id="streaming-transcode-size">
-          <h6>Maximum streaming transcode size</h6>
+          <h6>Maximum Transcode Size - Streaming</h6>
```

Comment:
This should at least be consistent with the change above:

`Maximum streaming transcode resolution`